netutils/config/parser.py

```diff
@@ -213,7 +214,7 @@ def _build_banner(self, config_line):
 
         raise ValueError("Unable to parse banner end.")
 
-    def _build_nested_config(self, line):
+    def _build_nested_config(self, line: str) -> t.Optional[str]:  # type: ignore
```

Comment:
If we stick a `return None` at the end of the function the behavior of the function should stay the same but we can get rid of the `# type: ignore`.